### PR TITLE
boards: arm: efm32gg_*: Enable watchdog, SWO and 72Mhz operation

### PR DESCRIPTION
**Pull request by thorvald**
This makes the EFM32GG11-based development kits use the 72Mhz HF RC-Oscillator for the CPU, rather than the 50Mhz HFXO crystal. The HFXO is only enabled if needed to drive the ethernet clock.
This also enables the watchdog, the TRNG and SWO output, and adds support for OpenOCD (but leaves the default as jlink).

Signed-off-by: Thorvald Natvig <thorvald@natvig.com>

### CODE REVIEW

**Comment by chrta** [comment]
@thorvald I have some issues testing this on my efm32gg_stk3701a. It seems something is broken with the ethernet driver (also on the zephyr master branch).I cannot get a lease with samples/net/dhcpv4_client. Does this work for you? It works on the nucleo_f429zi board.

File: dts/arm/silabs/efm32gg11b.dtsi [hunk 1]
```diff
@@ -288,6 +289,33 @@
 				#gpio-cells = <2>;
 				status = "disabled";
 			};
+
+
+			trng0: trng@4001d000 {
+				compatible = "silabs,gecko-trng";
+				reg = <0x4001d000 0x400>;
+				interrupts = <49 0>;
```
Comment (chrta) [inline dts/arm/silabs/efm32gg11b.dtsi:297]: 49 -> 66 see https://www.silabs.com/documents/public/reference-manuals/efm32gg11-rm.pdf

**Review by chrta — CHANGES_REQUESTED** [review]
Otherwise looks good to me. I successfully tested openocd and ethernet on efm32gg_stk3701a

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Please split the changes into 4 separate commits: watchdog, clocks, TRNG, and openocd. It's ok to keep in the same PR.

File: dts/arm/silabs/efm32gg11b.dtsi (hunk 1 above)
Comment (thorvald) [reply dts/arm/silabs/efm32gg11b.dtsi]: D'oh! Fixed.

**Comment by thorvald** [comment]
Split into independent commits.

**Review by chrta — APPROVED** [review]
LGTM

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**